src/Process/Deploy/InstallUpdate/ConfigUpdate/SearchEngine/Config.php

```diff
@@ -129,6 +137,10 @@ private function getElasticSearchConfiguration(array $config)
             $engine = 'elasticsearch' . intval($esVersion);
         }
 
+        if ($this->elasticSuite->isInstalled()) {
+            $engine = 'elasticsuite';
```

Comment:
in this case you ignore SEARCH_CONFIGURATION, isn't it?

Reply:
no, as it's merged later